helix-common/src/main/java/org/apache/helix/constants/InstanceConstants.java

```diff
@@ -8,4 +8,12 @@ public enum InstanceDisabledType {
     USER_OPERATION,
     DEFAULT_INSTANCE_DISABLE_TYPE
   }
+
+  public enum InstanceOperation {
+    EVACUATE,
+    SWAP_IN,
+    SWAP_OUT,
+    ENABLE,
+    DISABLE;
```

Comment:
Why we need both enable/disable... Suggest just have disable.

Reply:
TFTR. I suppose user can `enable` and disabled instance?

Reply:
Had an offline discussion.  We will keep `enable` and `disable` for backward compatibility.  When user set HELIX_ENABLED from false to true, we will set OPERATION to be `enabled` if the field is non empty and vice vesa.